src/main/java/org/ibp/api/java/impl/middleware/germplasm/GermplasmServiceImpl.java

```diff
@@ -491,4 +492,41 @@ void setCrossExpansionProperties(final CrossExpansionProperties crossExpansionPr
 		this.crossExpansionProperties = crossExpansionProperties;
 	}
 
+	@Override
+	public List<AttributeDTO> getGermplasmAttributes() {
```

Comment:
it maybe duplicated, read comment in the interface

Reply:
removed.